src/Orchard.Cms.Web/Startup.cs

```diff
@@ -36,6 +36,8 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddCommands();
 
+            services.AddAuthentication();
```

Comment:
> @PinpointTownes Has this sth to do with the fact that in your original PR you skip changes in oauth branch for Orchard.Users/Startup.cs file?

No, it doesn't. For some reasons, Orchard no longer registers the authentication services by default: since the data protection provider is not available when initializing the modules, the OpenID module crashes because its `Startup` can't be instantiated.

> Should we undo those changes from oauth branch?

No. They should be directly added to the master branch to ensure tenants are correctly isolated. I'll open a separate issue to track that.